Formula/treefrog.rb

```diff
@@ -0,0 +1,64 @@
+class Treefrog < Formula
+  desc "High-speed C++ MVC Framework for Web Application"
+  homepage "http://www.treefrogframework.org/"
+  url "https://github.com/treefrogframework/treefrog-framework.git", :tag => "v1.13.0", :revision => "dad8d4bbb4ae0bfb480352fde3ba5dd1321ccff8"
```

Comment:
Please split this onto multiple lines. Also, does this use submodules? If not, just use a tarball.